.github/ISSUE_TEMPLATE/2-feature.yml

```
@@ -1,4 +1,4 @@
-name: 🪴 Compiler Feature Request
+name: 🌻 Compiler Feature Request
```

Since this isn't part of the docs, i think it should be fine to undo this.
```suggestion
name: 🪴 Compiler Feature Request
```

---

Sure thing. Users like me are still going to see a `🯄` instead of the plant, but at least it's not comparable to the amount of visitors on the documentation.